text/0006-architecture-review-process.md

```diff
@@ -23,13 +23,13 @@ A **Design Review** is to decide if the thing should be done in a particular way
 
 For both a Roadmap Review and a Design Review the **timetable** is as follows:
 
-1. The team should lay the stage (see "initial meeting" below):
-  * Document the **question** (or questions) to be resolved, which should be agreed between the team asking for review and the chair.
+1. The team should set the stage (see "initial meeting" below):
+  * Summarize in a sentence or 2, the proposal to be reviewed. This should be agreed between the team asking for review and the chair.
   * Find someone to **chair** the review:
-    - For a Roadmap Review this will be the *least* senior manager with the ability to address the problem in its entirety.
+    - For a Roadmap Review this will be the least senior manager with the ability to address the problem in its entirety.
     - For a Design Review this will be an experienced engineer in the problem domain outside of the team willing to put time into the problem. As engineers don’t review their own code they shouldn’t review their own designs. The principle is to get the most informed and least biased feedback possible.
 
-2. The team produces a **Review Packet** designed to answer the question identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key questions and the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the answer are likely to be more significant.
+2. The team produces a **Review Packet** designed to document the proposal identified in step 1. The production of the Review Packet is expected to be collaborative and iterative to ensure that the review focuses on the key questions and the team presents its best case. This is particularly true of a Roadmap Review where the consequences of the answer are likely to be more significant.
```

Comment:
The rest of this sentence still talks about an answer, and I'm not sure that 'best case' is sufficiently collaborative. Perhaps switch the latter to something about clear reasoning, and change answer to 'decision' or 'outcome'?